DataConnectors/Okta Single Sign-On/AzureFunctionOktaSSO_V2/AzureFunctionOktaSSO/run.ps1

```diff
@@ -52,13 +52,20 @@ $TotalRecordCount = 0
 $apiToken = $env:apiToken
 $uri = $env:uri
 $StartDate = [System.DateTime]::UtcNow.ToString("yyyy-MM-ddT00:00:00.000Z") # set default fallback start time to 0:00 UTC today
+$logAnalyticsUri = $env:logAnalyticsUri
 
 # Define the Log Analytics Workspace ID and Key and Custom Table Name
 $customerId = $env:workspaceId
 $sharedKey =  $env:workspaceKey
 $LogType = "Okta"
 $TimeStampField = "published"
 
+# Returning if the Log Analytics Uri is in incorrect format.
+# Sample format supported: https://" + $customerId + ".ods.opinsights.azure.com
+if($logAnalyticsUri -notmatch 'https:\/\/([\w\-]+)\.ods\.opinsights\.azure.([\w\.]+)')
```

Comment:
This regex is not good enough, it will accept format with query strings. 

eg. https://<workspaceid>.ods.opinsights.auzre.com?api=....

This input will definitely fail the execution, but this input doesn't catch it.

Reply:
Updated regex which should not accept query strings and working fine with following scenarios.
https://" + $customerId + ".ods.opinsights.azure.com
https://" + $customerId + ".ods.opinsights.azure.us
https://" + $customerId + ".ods.opinsights.azure.scloud
https://" + $customerId + ".ods.opinsights.azure.eaglex.ic.gov
https://" + $customerId + ".ods.opinsights.azure.cn